qmlti/lib.php

```diff
@@ -217,9 +218,47 @@ function init_db($db) {
   [result_sourcedid] VARCHAR(255),
   [score] VARCHAR(255),
   [created] DATETIME,
+  [report_url] VARCHAR(255),
  CONSTRAINT [PK_{$outcome_table_name}] PRIMARY KEY CLUSTERED ([result_sourcedid] ASC, [score] ASC, [created] ASC)
 )
 EOD;
+ 
+//        $ok = $db->exec($sql) !== FALSE;
+        $db->exec($sql);
+
+      }
+
+      $reports_table_name = TABLE_PREFIX . LTI_Data_Connector::REPORTS_TABLE_NAME;
```

Comment:
The qmp_coaching_reports table is a 'tracker' to determine whether or not the coaching reports are able to be viewed by a participant. By doing so, we can simply update the table when the administrator wants to display coaching reports, and users will be able to access them.